packages/async-flow/src/async-flow.js

```diff
@@ -118,6 +118,7 @@ export const prepareAsyncFlowTools = (outerZone, outerOptions = {}) => {
           bijection, // membrane's guest-host mapping
           outcomeKit: makeVowKit(), // outcome of activation as host vow
           isDone: false, // persistently done
+          hostVowToCall: zone.detached().weakMapStore('hostVowToCall'),
```

Comment:
Besides being an ephemeral store, I think we shouldn't burden ourselves with weak collections and their gc costs since we have delimited lifetimes for these host call entries, and don't need to retain them past settlement. Since the replay membrane is keeping the result vow/promises live anyway, there is no reason to assume those might be dropped earlier either.